src/form.schema.json

```diff
@@ -93,6 +93,10 @@
               "$ref": "#/definitions/copy",
               "description": "The name of this group of sections"
             },
+            "instructions": {
```

Comment:
Added a new section called "instructions" to "sections"-type questions, it mirrors the way we implemented question "instructions".

As such, we have to add "instructions" to the sections-type form schema.